RockLib.UniversalMemberAccessor.Tests/TypeCreator/Define.cs

```diff
@@ -279,7 +279,7 @@ public static MemberDefinition WriteOnlyProperty(string name, Type type, bool is
             var il = getMethodBuilder.GetILGenerator();
 
             il.Emit(OpCodes.Ldstr, type.Name + " " + tb.Name + ".get_" + name + "()");
```

Comment:
Consider changing this to string interpolation

Reply:
Updated to string interpolation.